### ISSUE TITLE
Add docker instructions for developers in the docs

### ISSUE THREAD

**Issue by RemDelaporteMathurin** [ukaea/paramak#837]
Detailed instructions for contributors using Docker could be beneficial (which dependencies image should we use, which Docker command, etc...)

Maybe I'm just being lazy (or have a really bad memory) but I keep forgetting the syntax for running a container with shared volumes.

Plus, the ukaea/paramak:dependencies doesn't seem to be specified anywhere in the installation section.

Just adding the following command would save a lot of time (in my opinion): 

`docker run -it -v $(pwd):/home/shared ukaea/paramak:dependencies`

For windows:
`docker run -it -v ${PWD}:/home/shared ukaea/paramak:dependencies`

**Comment by shimwell** [comment]
yep nice suggestion. I can take a look at the docs for the install and perhaps move the install into a few separate sections

Everything is on the index page at the moment but that is a bit crowded
